packages/application-components/src/components/modal-page/README.md

```diff
@@ -0,0 +1,54 @@
+# ModalPage
+
+## Usage
+
+```js
+import { ModalPage } from '@commercetools-frontend/application-components';
+```
+
+#### Description
+
+Modal pages are controlled components used to render a page using a modal container, which makes it appear on top of the normal page. This is useful to render detailed data that normally requires some space.
+
+#### Usage
+
+```js
+<ModalPage
+  isOpen={true}
+  onClose={handleClose}
+  title="Lorem ipsus"
+  subtitle={<Text.Body>{'Lorem ipsus ...'}</Text.Body>}
+  components={{
+    actions: (
```

Comment:
What if we split and have to. So actions.primary and actions.secondary. While the modal retains control over position to limit the APIs flexibility.

Reply:
Yep, that's a good idea.